GestioneOrari/src/Elaborato/Elaborato.java

```diff
@@ -1,8 +1,23 @@
 package Elaborato;
 
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import row.Row;
+
 public class Elaborato {
+	private List<Row> rowsList;
 	
-	List<Commessa> commesse;
-	List<Giorni> giorno;
-
+	public Elaborato() {
+		rowsList = new ArrayList<>();
+	}
+	
+	public void addNewRowToRowsList(Date date, String startHour, String endHour, String order, String note) {
```

Comment:
Per add si intende che sia nuovo quindi puoi anche troncare il new dal nome e anche List non è necessario di solito quando si mette un plurale a un campo si tratta di una collection.

```suggestion
	public void addRowToRows(Date date, String startHour, String endHour, String order, String note) {
```